backend/ee/danswer/external_permissions/google_drive/doc_sync.py

```diff
@@ -123,12 +110,12 @@ def _fetch_permissions_paginated(
 def _fetch_google_permissions_for_document_id(
     db_session: Session,
     drive_file_id: str,
-    raw_credentials_json: dict[str, str],
+    credentials_json: dict[str, str],
     company_google_domains: list[str],
 ) -> ExternalAccess:
     # Authenticate and construct service
     google_drive_creds, _ = get_google_drive_creds(
```

Comment:
we let the get_google_drive_creds function determine the scopes (it already supports this)